recipes/cytominer_database/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "cytominer_database" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f253822e2e9c01e815492e06a6611405c4bb8daa615f24745c4308f41645ec10"
+
+build:
```

Comment:
This recipe may be able to be made `noarch: python`. Can you check the list in the docs?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
looks like it is able, I will fix